packages/protocol/contracts/common/FixidityLib.sol

```diff
@@ -80,16 +72,6 @@ library FixidityLib {
     return 115792089237316195423570985008687907853269984665640564;
   }
 
-  /**
-   * @notice Maximum value that can be safely used as an addition operator.
-   * @dev Test maxFixedAdd() equals maxUint256()-1 / 2
-   * Test add(maxFixedAdd(),maxFixedAdd()) equals maxFixedAdd() + maxFixedAdd()
-   * Test add(maxFixedAdd()+1,maxFixedAdd()+1) throws
-   */
-  function maxFixedAdd() internal pure returns (uint256) {
```

Comment:
Hmm, so I think this could still be a potentially useful number to keep in this file, at least in a comment. Can be useful information for consumers of this library.

Reply:
+1 to keeping as a comment